commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/pipeline/DbPipelineImpl.java

```diff
@@ -32,10 +32,10 @@ protected DbPipelineImpl() {
     }
 
     @Override
-    public List<Pipeline> getPipelinesForOrganization(String orgName) {
-        final String queryStr = "SELECT e FROM Pipeline e WHERE e.organization.name = :orgName";
+    public List<Pipeline> getPipelinesForOrganization(String organizationExternalId) {
```

Comment:
Name of the param for getPipelinesForOrganization should be aligned with interface method's param name